src/api/api.py

```diff
@@ -0,0 +1,34 @@
+from .pros import pros
+from flask import Flask, jsonify, abort
+from werkzeug.exceptions import HTTPException
+from flask_cors import CORS
+
+cors = CORS()
+
+api = Flask(__name__)
+
+cors.init_app(api)
+
+#error handlers
+@api.errorhandler(HTTPException)
+def http_error(error):
```

Comment:
Functions should typically be named as verbs. The name should indicate what the function does. `http_error()` < `handle_http_error()`.